cli/src/utils.ts

```diff
@@ -1,14 +1,15 @@
-import slugify from 'slugify';
-
 export function pluralize(word: string, count: number) {
   return `${word}${count === 1 ? '' : 's'}`;
 }
 
+// Applies to the slug too
+export const MAX_WORKSPACE_NAME_LENGTH = 55;
+
 // Based on `IsValidIdentifier` from xvalidator.go
```

Comment:
Non-blocking: Might be worth adding a tactical comment here linking to the very similiar (or identical?) function in frontend-next to protect future-us from accidentally letting them diverge